.github/workflows/osx.yml

```diff
@@ -0,0 +1,87 @@
+name: github-OSX
+
+on: [push, pull_request]
+#on:
```

Comment:
Are the comments a reminder for follow-on updates?

Reply:
Yes. I will use lines 4-8 in place of line 3 before merging.